templates/README.md

```
@@ -0,0 +1,446 @@
+#### Background Job Plugin
+```bash
+./create_bunkerweb_plugin.sh \
+  -d "Log analyzer with daily processing" \
+  --with-jobs \
+  --order 500 \
+  loganalyzer
+```
+
+## Generated Structure
+
+### Basic Plugin Structure
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)